libs/solid/profile/src/lib/services/profile-definition.service.ts

```diff
@@ -25,21 +26,174 @@ export class ProfileDefinitionService {
     @Inject(SOLID_CORE_CONFIG) private _config: SolidCoreConfig
   ) {}
 
+  //Now i created 2 ways to calling OpenAPI, one is for the 2.0 Version,
```

Comment:
Also ein "legacy"-mode?
Ich bin nicht sicher, ob sich das unbedingt gelohnt hat - wenn alles funktioniert kann Christian wahrscheinlich sehr schnell in allen Apps von 2.0 auf 3.0 wechseln. Aber ja - so ist es sicher.

Reply:
Ich hoffe, es ist nicht doppelt Arbeit wenn alles wieder zurückgebaut werden muss ;-)

Reply:
Genau. Ich behalte da einfach bis die anderen Apps auch die 3.0 hat.